tools.go

```diff
@@ -0,0 +1,9 @@
+//go:build tools
+// +build tools
+
+package tools
```

Comment:
Do we need this? I dont see us using build tags anywhere. But also fine with this just being left in too. Gonna approve it anw

Reply:
Probably don't need it but it will guarantee a consistent version for everything thanks to the go.mod file